Emby.Naming/TV/SeriesResolver.cs

```diff
@@ -12,7 +12,7 @@ public static partial class SeriesResolver
         /// <summary>
         /// Regex that matches strings of at least 2 characters separated by a dot or underscore.
         /// Used for removing separators between words, i.e turns "The_show" into "The show" while
-        /// preserving namings like "S.H.O.W".
+        /// preserving names like "S.H.O.W".
```

Comment:
I think I skipped over this in my prior PR because I was being conservative. This time around, I'm more confident that this change is correct.